lingua_franca/time.py

```diff
@@ -17,16 +17,26 @@
 from dateutil.tz import gettz, tzlocal
 
 
+__default_tz = None
+
+
+def set_default_tz(tz):
+    global __default_tz
+    # TODO tz validation/conversion from string
```

Comment:
the possible formats are a tzinfo object, or a standard string for timezone, like the mycroft.conf has. those are pretty universal strings to identify timezones and i think most users will prefer using those instead of tzinfo objects. i don't think we want fuzzy matching at all in here

i missed this TODO, will add this mapping to the PR and a custom exception with logs about proper usage